contrail-agent/hooks/contrail_agent_utils.py

```diff
@@ -197,15 +198,15 @@ def update_charm_status():
 
     changed = common_utils.apply_keystone_ca(ctx)
     changed |= common_utils.render_and_log("vrouter.env",
-        BASE_CONFIGS_PATH + "/common_vrouter.env", ctx)
+                                           BASE_CONFIGS_PATH + "/common_vrouter.env", ctx)
```

Comment:
i don't like such style... it would be better to revert style changes.

Reply:
Done